lib/src/main/java/uk/gov/di/ipv/cri/drivingpermit/library/config/ParameterStoreParameters.java

```diff
@@ -4,58 +4,56 @@
 
 public class ParameterStoreParameters {
 
-    public static final String DCS_HTTPCLIENT_TLS_CERT = "/dl-cri-api-v1/DCS/HttpClient/tlsCert";
+    public static final String DCS_HTTPCLIENT_TLS_CERT = "DCS/HttpClient/tlsCert";
 
-    public static final String DCS_HTTPCLIENT_TLS_KEY = "/dl-cri-api-v1/DCS/HttpClient/tlsKey";
+    public static final String DCS_HTTPCLIENT_TLS_KEY = "DCS/HttpClient/tlsKey";
 
     public static final String DCS_HTTPCLIENT_TLS_INTER_CERT =
-            "/dl-cri-api-v1/DCS/HttpClient/tlsIntermediateCertificate";
+            "DCS/HttpClient/tlsIntermediateCertificate";
```

Comment:
Just raised on @ElliotMurphyGDSs PR should these DCS ones not be set to the current values used for DCS as we dont intend to rename them